src/components/main/mainsection-blocks/mediaTable.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { FaTable, FaTimes, FaMinusCircle } from 'react-icons/fa';
+import gravityPsd from '../../../images/main-section/IMG1.jpg';
+import psdFoil from '../../../images/main-section/IMG2.jpg';
+import woodLogo from '../../../images/main-section/IMG3.jpg';
+import UniHeader from './UniHeader';
+
+const MediaTable = () => {
+  return (
+    <div className="main-section-block">
+      <UniHeader
+        mainLeftIco={<FaTable size="24px" />}
+        nameBlock="Media Table"
+        rightIco2={<FaTimes />}
+      />
+      <div className="main-section-block-section">
```

Comment:
do not make such huge components.
use hash or divide them into smaller components